mlflow/store/model_registry/databricks_workspace_model_registry_rest_store.py

```diff
@@ -28,3 +28,11 @@ def get_model_version_by_alias(self, name, alias):
             "'models:/model_name@alias_name', configure the MLflow client to target Unity Catalog "
             "and try again.",
         )
+
+    def _await_model_version_creation(self, mv, await_creation_for):
+        uc_hint = (
+            " For faster and more stable model version creation, consider using the Model "
```

Comment:
A random question. How much faster?

Reply:
TBH, we haven't done direct performance comparisons between control plane model registry copy speed and the data plane MLflow client upload speed. But we know that uploading large models via client to UC is fast while the workspace model registry copy may time out.